lib/FieldType/Mapper/TextLineFormMapper.php

```diff
@@ -44,4 +58,32 @@ public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, Field
                     ->setAutoInitialize(false)->getForm()
             );
     }
+
+    /**
+     * "Maps" Field form to current FieldType.
+     * Allows to add form fields for content edition.
+     *
+     * @param FormInterface $fieldForm Form for the current Field.
+     * @param FieldData $data Underlying data for current Field form.
+     */
+    public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
+    {
+        $fieldDefinition = $data->fieldDefinition;
+        $formConfig = $fieldForm->getConfig();
+        $label = $fieldDefinition->getName($formConfig->getOption('languageCode')) ?: reset($fieldDefinition->getNames());
+
+        $fieldForm
+            ->add(
+                $formConfig->getFormFactory()->createBuilder()
+                    ->create(
+                        'value',
+                        'text',
+                        ['required' => $fieldDefinition->isRequired, 'label' => $label]
```

Comment:
Unsure actually. There's a `maxlength` attribute on text input  but no `minlength` attribute so for min length bit more complicated. I guess you can skip my comment for now :)